### PR TITLE
updated keras models

### PR DESCRIPTION
**Pull request by gauravghati**
Improved Keras models performance and updated all the problems that I faced during practising your code

### CODE REVIEW

File: keras/klab-04-1-multi_input_linear_regression.py [hunk 1]
```diff
@@ -3,24 +3,27 @@
...
 import numpy as np
 
-x_data = [[73., 80., 75.],
+x_data = np.array([[73., 80., 75.],
           [93., 88., 93.],
           [89., 91., 90.],
           [96., 98., 100.],
-          [73., 66., 70.]]
-y_data = [[152.],
+          [73., 66., 70.]]);
```
Comment (kkweon) [inline keras/klab-04-1-multi_input_linear_regression.py]: Why is a semicolon needed?

File: keras/klab-04-1-multi_input_linear_regression.py [hunk 2]
```diff
@@ -3,24 +3,27 @@
...
+y_data = np.array([[152.],
           [185.],
           [180.],
           [196.],
-          [142.]]
+          [142.]]);
```
Comment (kkweon) [inline keras/klab-04-1-multi_input_linear_regression.py]: Same

File: keras/klab-04-1-multi_input_linear_regression.py (hunk 1 above)
Comment (gauravghati) [reply keras/klab-04-1-multi_input_linear_regression.py]: Ya, it is not needed. Actually, I'm a cpp coder and habitual to put ; at the end of each statement.

**Comment by kkweon** [comment]
Awesome! Thank you!

**Pull request merged**